frontend/__tests__/actions/edit-phone-number.test.tsx

```diff
@@ -1,40 +1,33 @@
 import { redirect } from '@remix-run/node';
 
-import { describe, expect, it, vi } from 'vitest';
+import { describe, expect, it } from 'vitest';
 
-import { action } from '~/routes/_gcweb-app.update-info';
+import { action } from '~/routes/_gcweb-app.personal-information.phone-number.edit';
 
-vi.mock('~/services/user-service.server.ts', () => ({
-  userService: {
-    getUserId: vi.fn(),
-    updateUserInfo: vi.fn(),
-  },
-}));
-
-describe('Update Information Action', () => {
+describe('Edit Phone Number Action', () => {
   afterEach(() => {
     vi.clearAllMocks();
     vi.resetModules();
   });
 
-  it('Should redirect if no validation errors', async () => {
+  it('Should redirect without validation erros', async () => {
```

Comment:
```suggestion
  it('Should redirect without validation errors', async () => {
```

Reply:
Just self-resolve when you fix it.. I don't need to re-review.

Reply:
Fixed the typo